versions.props

```diff
@@ -12,7 +12,8 @@ com.google.auto.service:auto-service-annotations = 1.0
 com.google.auto:auto-common = 1.1.2
 com.google.code.findbugs:findbugs-annotations = 3.0.1
 com.google.code.findbugs:jsr305 = 3.0.2
-com.google.dagger:* = 2.37
+com.google.dagger:* = 2.38.1
+com.google.devtools.ksp:symbol-processing-api = 1.5.21-1.0.0-beta07
 com.google.errorprone:* = 2.7.1
```

Comment:
is this the only version that solves the problem?
Can you add a comment as well please

Reply:
It's the [only available](https://repo.maven.apache.org/maven2/com/google/devtools/ksp/symbol-processing-api/) and newest version, because com.google.devtools.ksp was only added to Maven Central a few days ago. Once dagger does a new release and stops defaulting to the beta03 version which is only present on Google Maven, this can be removed.